1.2.cpp

```diff
@@ -0,0 +1,66 @@
+#define _USE_MATH_DEFINES
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+/*
+* \brief Вычисляет площадь грани куба
+* \param edge Длина ребра куба
+* \return Площадь грани куба
+*/
+
+double GetArea(double edge);
+
+/*
+* \brief Вычисляет площадь полной поверхности куба
+* \param edge Длина ребра куба
+* \return Площадь полной поверхности куба
+*/
+
+double GetSurfaceArea(double edge);
+
+/*
+* \brief Вычисляет объем куба
+* \param edge Длина ребра куба
+* \return Объем куба
+*/
+
+double GetVolume(double edge);
+
+int main()
+{
+	// объявление переменной edge типа double
+	double edge;
+	// вывод вспомогательной строки в консоль
+	cout << "Cube edge length: ";
+	// присваивание edge значение, введенного с консоли
+	cin >> edge;
+
+	// объявление констант area, SurfaceArea, volume, которым присвается значения функций  GetArea(edge), GetSurfaceArea(edge), GetVolume(edge) соответственно.
+	const double area = GetArea(edge);
+	const double SurfaceArea = GetSurfaceArea(edge);
+	const double volume = GetVolume(edge);
+
+	// вывод результатов
+	cout << "Area of the cube face: " << area << ", The total surface area of the cube: " << SurfaceArea << ", Cube volume: " << volume;
+
+	return 0;
+}
+
+// функция, которая принимает в качестве аргумента значение типа double и возвращает значение типа double.
+double GetArea(double edge)
+{
+	return edge * edge;
+}
+
+// функция, которая принимает в качестве аргумента значение типа double и возвращает значение типа double.
+double GetSurfaceArea(double edge)
+{
+	return edge * edge * 6;
+}
+
+// функция, которая принимает в качестве аргумента значение типа double и возвращает значение типа double.
+double GetVolume(double edge)
```

Comment:
Если параметры функции не изменяются, то они явно должны быть обозначены `const`
[By Alex Allain](https://www.cprogramming.com/tutorial/const_correctness.html)
>The const keyword allows you to specify whether or not a variable is modifiable. You can use const to prevent modifications to variables and const pointers and const references prevent changing the data pointed to (or referenced).
>Const дает вам возможность более четко документировать вашу программу и фактически обеспечивать соблюдение этой документации. Ключевое слово const обеспечивает вашим пользователям гарантии, позволяющие оптимизировать производительность без угрозы повреждения их данных, обеспечивая соблюдение вашей документации. Например, константные ссылки позволяют указать, что упомянутые данные не будут изменены; это означает, что вы можете использовать константные ссылки как простой и немедленный способ повышения производительности для любой функции, которая в настоящее время принимает объекты по значению, не беспокоясь о том, что ваша функция может изменить данные. Даже если это произойдет, компилятор предотвратит компиляцию кода и предупредит вас о проблеме. С другой стороны, если бы вы не использовали константные ссылки, у вас не было бы простого способа гарантировать, что ваши данные не были изменены.